ext/fetch/26_fetch.js

```diff
@@ -183,7 +193,8 @@ async function mainFetch(req, recursive, terminator) {
     req.method,
     req.currentUrl(),
     req.headerList,
-    req.clientRid,
+    req.client.rid,
+    req.client.options.allowHost,
```

Comment:
Instead of passing it in from JS to Rust on every call, just get/set it from the http client resource in Rust.